swarm/pss/pss.go

```diff
@@ -1034,3 +1047,10 @@ func (p *Pss) digestBytes(msg []byte) pssDigest {
 	copy(digest[:], key[:digestLength])
 	return digest
 }
+
+func checkAddress(addr *PssAddress) bool {
```

Comment:
Generally it doesn't make sense to accept pointer to byte slice, unless you want to modify the underlying pointer.

Reply:
Thanks I'll make a pass and audit this for this PR.

Reply:
should be the same way as proposed for `checkMsg`